bin/MadGraph5_aMCatNLO/cards/production/13TeV/GGToTauTau_Inclusive_M_50GeV_TuneCP5_madgraphLO_reweight/GGToTauTau_Inclusive_M_50GeV_TuneCP5_madgraphLO_reweight_run_card.dat

```diff
@@ -0,0 +1,174 @@
+  15.0	= bwcutoff ! (M+/-bwcutoff*Gamma)
+ #*********************************************************************
+ # Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+ # (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+ #*********************************************************************
+  False	= cut_decays ! Cut decay products 
+#*********************************************************************
+# Standard Cuts                                                      *
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+  0.0	= ptj ! minimum pt for the jets 
+  0.0	= ptb ! minimum pt for the b 
+  15.0	= ptl ! minimum pt for the charged leptons 
+  0.0	= misset ! minimum missing Et (sum of neutrino's momenta)
+  -1.0	= ptjmax ! maximum pt for the jets
+  -1.0	= ptbmax ! maximum pt for the b
+  -1.0	= ptlmax ! maximum pt for the charged leptons
+  -1.0	= missetmax ! maximum missing Et (sum of neutrino's momenta)
+  {}	= pt_min_pdg ! pt cut for other particles (use pdg code). Applied on particle and anti-particle
+  {}	= pt_max_pdg ! pt cut for other particles (syntax e.g. {6: 100, 25: 50}) 
+#
+# For display option for energy cut in the partonic center of mass frame type 'update ecut'
+#
+#*********************************************************************
+# Maximum and minimum absolute rapidity (for max, -1 means no cut)   *
+#*********************************************************************
+  -1.0	= etaj ! max rap for the jets 
+  -1.0	= etab ! max rap for the b
+  3.0	= etal ! max rap for the charged leptons 
+  0.0	= etabmin ! min rap for the b
+  0.0	= etalmin ! main rap for the charged leptons
+  {}	= eta_min_pdg ! rap cut for other particles (use pdg code). Applied on particle and anti-particle
+  {}	= eta_max_pdg ! rap cut for other particles (syntax e.g. {6: 2.5, 23: 5})
+#*********************************************************************
+# Minimum and maximum DeltaR distance                                *
+#*********************************************************************
+  0.0	= drbb ! min distance between b's 
+  0.0	= drbj ! min distance between b and jet 
+  0.0	= drjl ! min distance between jet and lepton 
+  0.0	= drbl ! min distance between b and lepton 
+  -1.0	= drbbmax ! max distance between b's
+  -1.0	= drbjmax ! max distance between b and jet
+  -1.0	= drjlmax ! max distance between jet and lepton
+  -1.0	= drblmax ! max distance between b and lepton
+#*********************************************************************
+# Minimum and maximum invariant mass for pairs                       *
+#*********************************************************************
+  0.0	= mmbb ! min invariant mass of a b pair 
+  50.0	= mmll ! min invariant mass of leptons 
```

Comment:
what's the reason to require mmll=50 here

Reply:
Keep similar generator cut as in standard DY samples (for example https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/dyellell012j_5f_NLO_FXFX/dyellell012j_5f_NLO_FXFX_run_card.dat#L130)